src/VisualStudio/Core/Test.Next/Services/ServiceHubServicesTests.cs

```diff
@@ -299,6 +299,8 @@ public async Task TestRemoteHostSynchronizeIncrementalUpdate(bool applyInBatch)
             var solution = Populate(workspace.CurrentSolution);
 
             // verify initial setup
+            await workspace.ChangeSolutionAsync(solution);
+            solution = workspace.CurrentSolution;
```

Comment:
prior code didn't represent how the workspace/oop actually work.  We were forking the solution, but then telling the workspace to sync it's workspace (even though we hadn't applied the solution back to the workspace).  This was working in the test code before because we were synthesizing a new workspace version every time we called UpdatePrimaryWorkspace.  I changed that to actually pass the real value.  But this means actually needing the real value to get updated.  Hence the above lines.